WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1153,9 +1153,6 @@ private extension ProductFormViewController {
     }
 
     func onAddEditDownloadsCompletion(data: ProductDownloadsEditableData) {
-        defer {
-            navigationController?.popViewController(animated: true)
-        }
```

Comment:
I think that we should keep this and remove the `popViewController` inside the `ProductDownloadListViewController` where is not needed.

Reply:
OK. Updated